Sources/UAPusher/Builder/Notification.swift

```diff
@@ -0,0 +1,44 @@
+import JSON
+
+/// Notification segment of an Urban Airship push notification payload
+/// See: https://docs.urbanairship.com/integration/#notfication-payload
+public final class Notification: BuildingBlock {
+    // MARK: Class fields
+    
+    /// Notification representation
+    var payload: JSON
+    
+    // MARK: Predefined values
+    
+    public enum Predefined: JSONRepresentable {
```

Comment:
Same comment. It can be valuable to have these test generators, but I don't think they belong here. It we want them, they should be separate from the core implementation of the block in my opinion.